moto/rds/responses.py

```diff
@@ -370,7 +377,7 @@ def modify_db_parameter_group(self) -> ActionResult:
     def describe_db_parameters(self) -> ActionResult:
         db_parameter_group_name = self.parameters.get("DBParameterGroupName")
         db_parameter_groups = self.backend.describe_db_parameter_groups(
-            {"name": db_parameter_group_name}
+            {"db_parameter_group_name": db_parameter_group_name}
```

Comment:
Good catch.  It's interesting that this wasn't picked up by our existing tests... This defect meant that calling `DescribeDBParameterGroups(DBParameterGroupName="group_name")` was the same as calling `DescribeDBParameterGroups()` with no parameters, which only worked because we were only ever dealing with one DBParameterGroup in our tests.  Anyway, thanks for running into this and fixing it!